library/crux

```diff
@@ -1,4 +1,16 @@
-# maintainer: James Mills <prologic@shortcircuit.net.au> (@prologic)
+Maintainers: James Mills <prologic@shortcircuit.net.au> (@prologic)
```

Comment:
With all due respect to James, he has not been an active maintainer for quite some time now (which is the whole reason this image was deprecated).  Is this going to be a one-off drive-by update, or real long-term maintenance of the `crux` image?

Reply:
I have re-written how Docker Crux is handled to make it trivial for James to release new versions:

`./release build <version>`
`./release commit <version>`
`./release test <version>`
`./release push <version>`
`./release library`

He is now engaged again.

Reply:
Also, I'm happy to stay on as a Member and will back James up if he is too busy.